projboard/tests/test_article.py

```diff
@@ -50,20 +15,18 @@ def test_part_functions(self, create_articles):
         assert article.part_of_title(100) == article.title[:100]
         assert article.part_of_content(100) == article.content[:100]
 
-    @pytest.mark.django_db
-    def test_object_saved_in_db(self, create_articles):
-        assert create_articles[0] in Article.objects.all()
-        assert create_articles[1] in Article.objects.all()
+    def test_object_saved_in_db(self, articles):
+        for i in range(len(articles)):
+            assert articles[i] in Article.objects.all()
 
-    @pytest.mark.django_db
-    # using Edite function
-    def test_edit_object(self, create_articles, create_subjects):
+    # using Edit function
+    def test_edit_object(self, articles, subjects):
 
-        article = create_articles[0]
+        article = articles[0]
```

Comment:
Sorry, I see it only now :)
What does this method should do 
def test_edit_object
Which object? consider edit_article 

Dont you need a method in article? inorder to test changes? 
https://github.com/redhat-beyond/Smarticle/blob/main/projboard/models/article.py#L40

Reply:
Updated, considered just editing `article` with `edit` function